app/models/validations/tenancy_validations.rb

```diff
@@ -23,7 +23,7 @@ def validate_general_needs_fixed_tenancy_length_affordable_social_rent(record)
     min_tenancy_length = 2
     return if record.tenancylength.to_i.between?(min_tenancy_length, 99)
 
-    message = I18n.t("validations.tenancy.length.invalid_fixed", min_tenancy_length:)
+    message = I18n.t("validations.lettings.tenancy.length.invalid_fixed", min_tenancy_length:)
```

Comment:
Same here?
`I18n.t("validations.lettings.tenancy.needstype.invalid_fixed_tenancylength", min_tenancy_length:)` etc

Reply:
Sure

Reply:
Done. Is this so that we can have different messages for each field?